### ISSUE TITLE
mime type icons are not associated for .mmp/.mmpz on ubuntu

### ISSUE THREAD

**Issue by mikobuntu** [LMMS/lmms#1063]
Running latest git stable-1.1 ( this has been a long standing bug as far as i can remember on ubuntu and possibly other *nix distros ) and having LMMS installed system wide i do not get the application-x-lmms-project.svg file to show for any .mmp or .mmpz files until i manually copy the svg file to the proper location and update the cache again. here is how i accomplished this https://help.ubuntu.com/community/AddingMimeTypes

_edit .xpf files also now use the svg icon too, which is not a bad thing, but i however suggest we use the honeycomb coloured icon for these, which will make them easier distinguished ( i can file another issue/ticket for this if need be )_

**Comment by tresf** [comment]
> i do not get the application-x-lmms-project.svg file to show for any .mmp or .mmpz files until i manually copy the svg file to the proper location and update the cache again

@mikobuntu You'll still need to force-update the cache, but this should be addressed once #3688 is merged via eb5e819.
